diff --git a/src/sugar3/activity/bundlebuilder.py b/src/sugar3/activity/bundlebuilder.py
index 11e6edf..ff4bdbd 100644
--- a/src/sugar3/activity/bundlebuilder.py
+++ b/src/sugar3/activity/bundlebuilder.py
@@ -37,7 +37,7 @@ from sugar3 import env
 from sugar3.bundle.activitybundle import ActivityBundle
 
 
-IGNORE_DIRS = ['dist', '.git', 'screenshots']
+IGNORE_DIRS = ['dist', '.git', 'screenshots', 'debian', '.pc']
 IGNORE_FILES = ['.gitignore', 'MANIFEST', '*.pyc', '*~', '*.bak', 'pseudo.po']
 
 
